docs-zh/teams/ci.md

```diff
@@ -2,6 +2,93 @@
 
 `envd` 仅有一个依赖软件：[Buildkitd](https://github.com/moby/buildkit#containerizing-buildkit)。因此 `envd` 兼容绝大多数持续集成系统。如果你在这方面遇到问题，可以通过 [💬 Discord](https://discord.gg/KqswhpVgdU) 联系我们。我们很乐于提供这方面的帮助。
 
+## Deamonless Mode
+
+如果你想像 Kaniko 一样，在一个 Job 中运行 envd 构建镜像的过程，那么你可以使用 `tensorchord/envd:${ENVD_VERSION}-rootless` 镜像：
```

Comment:
Maybe we can add the pros/cons of this feature.

Reply:
SGTM